src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -453,4 +453,13 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="ReferenceCountSingular" xml:space="preserve">
     <value>1 reference</value>
   </data>
+  <data name="CodefixEncounteredError" xml:space="preserve">
+    <value>Codefix '{0}' encountered an error and has been disabled.</value>
```

Comment:
I am not sure 'Codefix' is a term end user is familiar with? Also need to consider refactoring.

If we want to say Codefix then I think we need a space i.e. 'Code Fix'.